components/os/unix.go

```diff
@@ -45,6 +53,14 @@ func getDefaultInstanceType(env config.Environment, arch Architecture) string {
 }
 
 func getUnixInstallFormatString(scriptName string, version AgentVersion) string {
```

Comment:
I think it would be great to do some refactoring here to make this function accept distro type as an argument. It should be possible to also use this on RPM-based platforms (either RH-based or SUSE-based) with `TESTING_YUM_URL` and `TESTING_YUM_VERSION_PATH`.